lib/pages/chat/chat.dart

```diff
@@ -397,28 +398,25 @@ class ChatController extends State<Chat> {
     );
   }
 
-  Future<void> sendImage(IndexedAssetEntity entity) async {
-    final matrixFile = await entity.toMatrixFile();
-    if (matrixFile != null) {
-      await room!.sendFileEvent(
-        matrixFile,
-        thumbnail: null,
-      ).catchError((e) {
-        Fluttertoast.showToast(
-          msg: "error: $e",
-          gravity: ToastGravity.BOTTOM,
-        );
-        return null;
-      });
+  void sendImage() {
+    final assetEntity = imagePickerController.selectedAssets.first;
+    final sendImageInteractor = getIt.get<SendImageInteractor>();
+    if (assetEntity.asset.type == AssetType.image) {
+      sendImageInteractor.execute(room: room!, entity: assetEntity.asset);
+      imagePickerController.clearAssetCounter();
+    numberSelectedImagesNotifier.value = 0;
     }
   }
 
-  Future<void> sendImages() async {
-    final selectedAssets = imagePickerController.sortedSelectedAssets;
-    for (final entity in selectedAssets) {
-      await sendImage(entity);
-    }
-  }
+  // Future<void> sendImages() async {
+  //   final selectedAssets = imagePickerController.sortedSelectedAssets;
+  //   for (final entity in selectedAssets) {
+  //     await sendImage();
+  //   }
+
+  //   imagePickerController.clearAssetCounter();
+  //   numberSelectedImagesNotifier.value = 0;
+  // }
```

Comment:
Remove it?

Reply:
I think i will use it for next ticket about send multiple images